client/src/components/PhotoCarousel.jsx

```diff
@@ -15,18 +16,34 @@ export default class PhotoCarousel extends Component {
         'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/redsky.jpg',
         'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/sandy-shores.jpg',
         'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/tree-of-life.jpg',
-        'https://images.unsplash.com/photo-1548678967-f1aec58f6fb2?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2090&q=80',
-        'https://images.unsplash.com/photo-1548625149-720134d51a3a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/aurora.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/city.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/mountains.jpg',
+        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/redsky.jpg',
       ],
+      showModal: false,
     };
   }
 
+  handleClick = () => {
```

Comment:
We can consider using one function for changing the state of showModal.

const { showModal } = this.state;

const newStatus = !showModal;

this.setState({
  showModal: newStatus
})